src/js/paste.js

```diff
@@ -145,7 +145,7 @@ var PasteHandler;
                     }
                 }
             } else {
-                this.pasteHTML(text);
+                this.pasteHTML(text, this.base.options);
```

Comment:
agree. I saw this ticket come in and wanted to suggest the exposure of those options be in a "paste" member (like my doc alludes to) ... eg:

```
new Editor(...., {
     extensions: {
          "paste":{
                 filterRegexp: [ /* additional regexps desired */ ],
                 cleanTags:[....], whateverTheOptionNameIs:[]
          }
     }
});
```